### DISCUSSION TITLE
New User Creation sends Cloud Welcome Email rather than Self-Hosted welcome email template.

### DISCUSSION THREAD

**Discussion by kblattman1** [Budibase/budibase#6516] [Help]
When I onboard a new user, they're sent the proper Invitation Email from our self-hosted instance, but shortly after, they get a Welcome email directing them to the cloud instance. I've attached some pictures to hopefully explain better:

The Invitation email they receive, with the 'Set up account' link taking them to our self-hosted instance.
![tempsnip](https://user-images.githubusercontent.com/108492851/176773480-bbcbf5fa-c3bf-4af4-b82c-19e576fac950.png)

The welcome email they receive, the 'Get Started' link sending them to account.budibase.app
![image](https://user-images.githubusercontent.com/108492851/176773042-bb53992e-ebef-4f48-97a5-64ea53f576ed.png)

How can I get this set up to use the "User Welcome" template in the Email section in the Manage section instead of the one pictured above to redirect them to our self-hosted instance instead of the cloud instance?

**Comment by melohagan** [comment]
HI @kblattman1 

I think the email above is a marketing email as I can't find that text anywhere within the codebase, so that email won't be editable.

I'll pass this discussion on so that the team is aware of this scenario.  

An issue has also been raised here: https://github.com/Budibase/budibase/issues/6520
